src/tests/test_api.py

```diff
@@ -9,6 +11,9 @@
 
 from app import app
 
+HOST_PORT_RE = re.compile(r"[a-zA-Z0-9.-]{3,63}:[0-9]{2,5}")
```

Comment:
No single-digit port numbers allowed, ey?  :raised_eyebrow:   :wink:

Reply:
Probably not gonna happen? Although apparently 4, 6, and 8 are free: https://en.wikipedia.org/wiki/List_of_TCP_and_UDP_port_numbers

Reply:
That's fine i agree probably not.

Well not until someone sees this check and decides to be a stinker about it :)